target/arm/translate-sve.c

```diff
@@ -6386,3 +6386,19 @@ static bool trans_UABA(DisasContext *s, arg_rrr_esz *a)
 {
     return do_sve2_fn3(s, a, arm_gen_gvec_uaba);
 }
+
+#define DO_SVE2_ZPZZ_FP(NAME, name) \
+static bool trans_##NAME(DisasContext *s, arg_rprr_esz *a)                \
+{                                                                         \
+    static gen_helper_gvec_4 * const fns[4] = {                           \
+        NULL,                            gen_helper_sve2_##name##_zpzz_h, \
+        gen_helper_sve2_##name##_zpzz_s, gen_helper_sve2_##name##_zpzz_d, \
+    };                                                                    \
+    return do_sve2_zpzz_ool(s, a, fns[a->esz]);                           \
+}
```

Comment:
Because of the status argument that the helpers need, you won't be able to directly re-use do_sve2_zpzz_ool. 
Compare do_zpzz_fp, which adds the get_fpstatus_ptr() argument.

Reply:
Looks good, thanks.